test/models/driver_test.rb

```diff
@@ -68,14 +68,28 @@
       # Your code here
     end
 
-    describe "can go online" do
-      # Your code here
-    end
+    describe "can go on/off line" do
 
-    describe "can go offline" do
-      # Your code here
+      it "can access the availability path" do
+        new_driver.save
+        # Act
+        patch availability_path(new_driver.id)
```

Comment:
This line produces an error.

The route you define is:
`patch 'drivers/availability', to: 'drivers#availability', as: 'availability'`
I think you meant to include the driver.id as a param:
`patch 'drivers/:driver_id/availability', to: 'drivers#availability', as: 'availability'`


```
  test_0001_can access the availability path                     ERROR (0.00s)
Minitest::UnexpectedError:         NoMethodError: undefined method `availability_path' for #<Minitest::Rails::SpecTests::Test__Driver_custom_methods_can_go_on_off_line__test_models_driver_test_rb__71:0x00007f92c0c88688>
            test/models/driver_test.rb:76:in `block (4 levels) in <main>'
```